### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DUT-lujunyu**
Add *The hateful memes challenge: Detecting hate speech in multimodal memes*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -652,7 +652,19 @@ nt-dates/data/](https://amiibereval2018.wordpress.com/important-dates/data/)
 * Platform: Twitter 
 * Medium: Text 
 * Reference: Ousidhoum, N., Lin, Z., Zhang, H., Song, Y. and Yeung, D., 2019. Multilingual and Multi-Aspect Hate Speech Analysis. ArXiv,. 
-  
+
+#### The Hateful Memes Challenge: Detecting Hate Speech in Multimodal Memes
+* Link to publication: [https://arxiv.org/pdf/2005.04790.pdf](https://arxiv.org/pdf/2005.04790.pdf)
+* Link to data: [https://www.drivendata.org/competitions/64/hateful-memes/](https://www.drivendata.org/competitions/64/hateful-memes/)
```
Comment (leondz) [inline README.md:658]: this should link to the dataset direct download

Comment (DUT-lujunyu) [reply README.md:658]: Ok, I will submit a new one~

File: README.md [hunk 2]
```diff
@@ -655,7 +655,7 @@ nt-dates/data/](https://amiibereval2018.wordpress.com/important-dates/data/)
 
 #### The Hateful Memes Challenge: Detecting Hate Speech in Multimodal Memes
 * Link to publication: [https://arxiv.org/pdf/2005.04790.pdf](https://arxiv.org/pdf/2005.04790.pdf)
-* Link to data: [https://www.drivendata.org/competitions/64/hateful-memes/](https://www.drivendata.org/competitions/64/hateful-memes/)
+* Link to data: [https://hatefulmemeschallenge.com/](https://hatefulmemeschallenge.com/) Get complete data after filling out the form.
```
Comment (leondz) [inline README.md:658]: I get an https security warning for this URL, do you get one too?

Comment (DUT-lujunyu) [reply README.md:658]: yes, but we can still visit it with advanced settings.

Comment (leondz) [reply README.md:658]: ok, let's move forward once that's fixed and the https works properly

**Comment by DUT-lujunyu** [comment]
您好，您的邮件已经收到，谢谢

**Pull request closed without merging**